cmake/alpakaCommon.cmake

```diff
@@ -141,6 +141,10 @@ if(NOT TARGET alpaka)
     add_library(alpaka INTERFACE)
 
     target_compile_features(alpaka INTERFACE cxx_std_${alpaka_CXX_STANDARD})
+
+    # Force MSVC to set a correct version macro: https://devblogs.microsoft.com/cppblog/msvc-now-correctly-reports-__cplusplus/
+    target_compile_options(alpaka INTERFACE $<$<CXX_COMPILER_ID:MSVC>:"/Zc:__cplusplus">)
+
```

Comment:
Why is this necessary?

Reply:
It is a remnant from when I originally approached this in the way described above (i.e., checking for `__cplusplus` etc.). I'll remove this since it seems to break MSVC for some weird reason.